main.go

```diff
@@ -42,6 +43,10 @@ func main() {
 		}(item.ConnType, item.Connection, dmuxLogging.EnableDebug)
 	}
 
+	go func() {
+    	log.Println(http.ListenAndServe("localhost:8081", nil))
+    }()
+
```

Comment:
metrics is also being exposed on a port, can we use the same port? 


Also is p8s format metrics being published?